apps/common/translations/src/en/pages.json

```diff
@@ -0,0 +1,11 @@
+{
+  "start": {
+    "header": "Illegal migration act"
+  },
+  "declaration": {
+    "header": "Declaration"
+  },
```

Comment:
Think we can remove these from common - they are already in the IMA app.

Reply:
Removed